articles/active-directory/develop/quickstarts/active-directory-android.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Azure AD v2 Android quickstart | Microsoft Docs
+description: Learn how Android native applications can call an API that requires access tokens by Azure Active Directory v2.0 endpoint.
+services: active-directory
+documentationcenter: dev-center-name
+author: andretms
+manager: mtillman
+editor: ''
+
+ms.assetid: 820acdb7-d316-4c3b-8de9-79df48ba3b06
+ms.service: active-directory
+ms.component: develop
+ms.devlang: na
+ms.topic: article
+ms.tgt_pltfrm: na
+ms.workload: identity
+ms.date: 07/20/2018
+ms.author: andret
+ms.custom: aaddev 
+#Customer intent: As an application developer, I want to learn how Android native apps can call an API that requires access tokens by Azure AD v2.0 endpoint.
+---
+
+# Quickstart: Sign in users and call the Microsoft Graph API from an Android app
+
+This quickstart contains a code sample that demonstrates how a native Android application can sign in personal, work and school accounts, get an access token, and call the Microsoft Graph API.
```

Comment:
I think there may be a grammatical issue here, my understanding of the branding is:

- Personal
- Work and School

So I think it would be: Android app can sign in personal or work and school accounts.

Reply:
May be worth linking to a description of these account types. What is a Microsoft account and what is an Azure AD account. Note, this article may not exist at this time, but may be worht considering.